docs/developing/connect_rpc.md

```diff
@@ -4,10 +4,11 @@ proofedDate: 20230526
 iterationBy: na
 includedInSite: true
 approvedBy: na
-comment: 
+comment: todo Find out from Yury whether we can have a NeonLabs youtube domain to support the video material that has been created to complement this page
```

Comment:
@YYYurchenko I created video material that supports this page. Once it is clear who has mandate for docs pages, if it is still product, then I want to link to that video support